### ISSUE TITLE
Package can't be imported with version (only using branch)

### ISSUE THREAD

**Issue by DarthMike** [WalletConnect/WalletConnectSwiftV2#567]
**Describe the bug**
A package can be imported through SPM using a branch or a version number. 
The tags in the repository are currently of the form `vx.y.z`, so adding the package is impossible using a specific version.

**SDK Version**
 - Client: Xcode/Swift pacakge

**To Reproduce**
1. Create new Xcode project
2. Add a package dependency in the UI
3. Enter the repo url
4. See the last available version is 0.6.0

Same step to reproduce if building a package. In `package.swift`:
```swift
dependencies: [
.package(url: "https://github.com/WalletConnect/WalletConnectSwiftV2", .exact("1.0.4"))
]
```

**Expected behavior**
Should be able to import package by version number

**Screenshots**
![Screenshot 2022-11-03 at 14 26 39](https://user-images.githubusercontent.com/1030987/199732950-a601e511-8748-487a-b679-3ceeb6d9a191.png)

**Device (please complete the following information):**
Xcode

**Additional context**
N/A

**Comment by llbartekll** [comment]
hey, @DarthMike it is fixed now, you can try last exact version 1.0.4

**Comment by DarthMike** [comment]
Thanks @llbartekll
